Jellyfin.Server.Implementations/Migrations/20201203203707_AddCustomDisplayPreferences.cs

```diff
@@ -0,0 +1,70 @@
+﻿#pragma warning disable CS1591
+// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Jellyfin.Server.Implementations.Migrations
+{
+    public partial class AddCustomDisplayPreferences : Migration
```

Comment:
We don't want a _new_ migration. We have to edit the old one, otherwise this data will be lost from 10.6.z -> 10.7.

Reply:
I can edit the db migration, but this is the EFCore migration to actually add the table

Reply:
@crobibero Isn't that a matter of replacing the code of the old one with the code in the new one?

Reply:
My interpretation was to keep the migrated preferences, and just store the "custom" ones in a separate table.
If we would prefer to just store all of the preferences in a dictionary I can change this PR

Reply:
@crobibero Oh no that's fine, in fact I think it's the best approach. My fault for not understanding well what you wanted to accomplish.